docs/streams/stream-tasks/view-streams.md

```diff
@@ -20,7 +20,118 @@ View existing streams.
 
 1. To see more details and stream statistics, click a stream name.
 
-When you view a stream in the console, you will not see a backlog of messages. This screen does not persist, which means that when it is open, you will see new messages as they are sent. If you close the screen, any messages displayed will not be visible the next time that you open it.
+</TabItem>
+<TabItem value="py" label="Python SDK">
+
+View a list of existing streams.
+
+```python
+from c8 import C8Client
+
+""" For Python SDK, you can omit https:// part of the URL """
+BASE_URL = "play.paas.macrometa.io/"
+
+""" Connect to GDN """
+client = C8Client(
+    protocol='https',
+    host=BASE_URL,
+    port=443,
+    apikey='<your API key>', # Enter your API key
+    geofabric="<your-fabric>" # Enter your fabric name
+)
+
+streams = client.get_streams()
+
+print("List of streams:")
+for stream in streams:
+    print(f"- Name: {stream['name']}")
+    print(f"  Topic: {stream['topic']}")
+    print(f"  Type: {stream['type']}")
+    print(f"  Status: {stream['status']}")
+```
+
+Get stats for a specific stream.
+
+```python
+import json
+from c8 import C8Client
+
+# Define variables
+URL = "play.paas.macrometa.io"
```

Comment:
This structure for creating clients is not the same as the above py example; they both correct but I think we should stick with one

Reply:
I don't understand what you mean, here.

Reply:
We have two Python examples on the same page but we are using slightly different approach when we create "client" object.
For example in the first one we dont have a variable API key but in the second one we have that variable (API_KEY = "your-api-key")

Reply:
Gotcha. Content updated, please review.